### PR TITLE
[0518] Add ancient languages to list of secondary subjects

### PR DESCRIPTION
**Pull request by defong**
### Context
Ancient languages

### Changes proposed in this pull request
Add ancient languages to list of secondary subjects, Latin, Ancient Greek & Ancient Hebrew, with 15k bursary

### Guidance to review
This change only affects downstream 
The below is tied to this PR

#### Find
https://find-pr-1150.london.cloudapps.digital/start/subject?l=2

image

#### Publish
https://publish-teacher-training-pr-2063.london.cloudapps.digital/

Create a course, with one of the following secondary subjects, Latin, Ancient Greek & Ancient Hebrew
ie https://publish-teacher-training-pr-2063.london.cloudapps.digital/organisations/T92/2022/courses

image

### Checklist

- [x] Make sure all information from the Trello card is in here
- [x] Attach to Trello card
- [x] Rebased master
- [x] Cleaned commit history
- [x] Tested by running locally

### CODE REVIEW

**Comment by defong** [comment]
[Add Latin, Ancient Greek, and Ancient Hebrew to list of Secondary Subjects](https://trello.com/c/ULa4VlZt/518-add-latin-ancient-greek-and-ancient-hebrew-to-list-of-secondary-subjects)

File: db/seeds.rb [hunk 1]
```diff
@@ -23,6 +23,8 @@
 Subjects::FinancialIncentiveCreatorService.new(year: 2021).execute
 Subjects::FinancialIncentiveSetSubjectKnowledgeEnhancementCourseAvailableService.new(year: 2021).execute
 
+Subjects::FinancialIncentiveCreatorService.new(year: 2022).execute
```
Comment (stellenberger) [inline db/seeds.rb:26]:
```suggestion
Subjects::FinancialIncentiveSetSubjectKnowledgeEnhancementCourseAvailableService.new(year: 2021).execute

[2021, 2022].each do |year|
  Subjects::FinancialIncentiveCreatorService.new(year: year).execute
end
```

**Review by stellenberger — APPROVED** [review]
Looks good, just a few cosmetic comments

**Review by mbocc — APPROVED** [review]
Have reviewed - We can now add a new course with an ancient language as its subject (Latin, ancient Hebrew, Ancient Greek), as well as edit an existing course to one of those subjects. 

All approved - Thanks for this!

File: app/services/subjects/financial_incentive_creator_service.rb [hunk 2]
```diff
@@ -8,6 +8,11 @@ def initialize(year:, subject: Subject, financial_incentive: FinancialIncentive)
 
     def subject_and_financial_incentives
       subject_and_financial_incentives = {
+        2022 => {
+          ["Latin", "Ancient Greek", "Ancient Hebrew"] => {
```
[the remark this answers is not in the document]
Comment (stellenberger) [reply app/services/subjects/financial_incentive_creator_service.rb:12]: Actually, not viable as whitespace will cause issue. Scrub that!

**Pull request merged**